wdio.conf.js

```diff
@@ -0,0 +1,45 @@
+/* global browser */
+/* eslint-disable import/no-extraneous-dependencies */
+const wdioConf = require('terra-toolkit/lib/wdio/conf');
+const WebpackDevService = require('terra-toolkit/lib/wdio/services/index').WebpackDevService;
+const localIP = require('ip');
+const path = require('path');
+const webpackConfig = require('./packages/terra-site/webpack.config.js');
+
+const webpackPort = 8080;
+
+// Flex specs search between local pacakge and repo
+let specs = path.join('tests', 'wdio', '**', '*-spec.js');
+if (__dirname === process.cwd()) {
+  specs = path.join('packages', '**', specs);
+}
+
+const config = {
+  ...wdioConf.config,
+
+  baseUrl: `http://${localIP.address()}:${webpackPort}`,
+  specs,
+
+  seleniumDocker: {
+    enabled: !process.env.TRAVIS,
+  },
+
+  webpackPort,
+  webpackConfig,
+  before() {
```

Comment:
I like the idea of this as a candidate for toolkit. @bjankord @ryanthemanuel what are your thoughts? This before enables us to ensure the themeable variables are correctly applied. 

If we want to wait on this as a toolkit helper, I would be in favor of breaking this before into its own file. At minimal documentation describe the before func.

Reply:
If we broke it out, where would we put it?